backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/PostgresDocumentationUnitSearchRepositoryImpl.java

```diff
@@ -254,6 +256,24 @@ private List<Predicate> getMyDocOfficePredicates(
     return predicates;
   }
 
+  private List<Predicate> getAssignedToMePredicates(
+      SearchParameters parameters,
+      HibernateCriteriaBuilder cb,
+      Root<DocumentationUnitDTO> root,
+      OidcUser oidcUser) {
+    List<Predicate> predicates = new ArrayList<>();
+    User user = userService.getUser(oidcUser);
```

Comment:
Das können wir zwecks Sparsamkeit in die nächste Zeile verschieben, oder?

Reply:
oh ja gut gesehen!